README.adoc

```diff
@@ -59,6 +59,23 @@ Library], a collection of reusable, battle-tested, production ready infrastructu
 * link:/core-concepts.md#what-is-helm[What is Helm?]
 * _https://www.manning.com/books/kubernetes-in-action[Kubernetes in Action]_: the best book we've found for getting up and running with Kubernetes.
 
+=== Get started
+* link:/charts/k8s-service/README.md##how-to-use-this-chart[How to use this chart?]
+* link:/charts/k8s-service/README.md#what-resources-does-this-helm-chart-deploy[What resources does this Helm Chart deploy?]
+* link:/charts/k8s-service/README.md#how-do-i-deploy-additional-services-not-managed-by-the-chart[How do I deploy additional services not managed by the chart?]
+* link:/charts/k8s-service/README.md#how-do-i-expose-my-application-internally-to-the-cluster[How do I expose my application internally to the cluster?]
+* link:/charts/k8s-service/README.md#how-do-i-expose-my-application-externally-outside-of-the-cluster[How do I expose my application externally outside of the cluster?]
+* link:/charts/k8s-service/README.md#how-do-i-deploy-a-worker-service[How do I deploy a worker service?]
+* link:/charts/k8s-service/README.md#how-do-i-check-the-status-of-the-rollout[How do I check the status of the rollout?]
+* link:/charts/k8s-service/README.md#how-do-i-set-and-share-configurations-with-the-application[How do I set and share configurations with the application?]
+* link:/charts/k8s-service/README.md#how-do-you-update-the-application-to-a-new-version[How do you update the application to a new version?]
+* link:/charts/k8s-service/README.md#how-do-i-verify-my-canary-deployment[How do I verify my canary deployment?]
+* link:/charts/k8s-service/README.md#how-do-i-roll-back-a-canary-deployment[How do I roll back a canary deployment?]
+* link:/charts/k8s-service/README.md#how-do-i-ensure-a-minimum-number-of-pods-are-available-across-node-maintenance[How do I ensure a minimum number of Pods are available across node maintenance?]
+* link:/charts/k8s-service/README.md#why-does-the-pod-have-a-prestop-hook-with-a-shutdown-delay[Why does the Pod have a preStop hook with a Shutdown Delay?]
+* link:/charts/k8s-service/README.md#what-is-a-sidecar-container[What is a sidecar container?]
+* link:/charts/k8s-service/README.md#how-do-i-use-a-private-registry[How do I use a private registry?]
```

Comment:
NIT: some of these are already linked down below in the manage section, which is sort of where they belong in the current format. We can discuss if we should do a format change (I do slightly prefer this format), but we should hold off on making those changes until we have decided this so that our repos can be consistent.

Reply:
Okay, is it better to revert adding all the links here, then? So there won't be any direct links to most of the stuff in core-concepts.md?

Reply:
Sorry should have been more clear on the suggestion. If you agree with what I said, then we should put all these links in the `Manage` section down below in the document.

Reply:
If you're not strongly opinionated on this, could we start doing this, where we link to everything in core-concepts? So, keep this change, but remove the duplicated links that are already in `manage`.

Reply:
I'll push up a change and see what you think. A bit of reorg.

Reply:
I am not strongly opinionated on whether we should do this format change, but I am strongly opinionated that a format change like this should be a team decision and we should go through our regular process to make those team changes. I care a lot about having a consistent look to our docs and having the change isolated in this repo moves counter to that.

Given that, here is a middle ground proposal:

- Keep this section.
- We remove the manage section.
- Before we merge this, start a discussion somewhere talking about this format change. Either an RFC in `team-product` or a `/raise` in slack. Once the discussion is kicked off, we can merge this without waiting on the results (it should be fairly easy to revert it should we decide against it). The reason to gate the merge on starting the discussion is to make sure we have started the discussion before we forget that we did this.

Does that sound good?

Reply:
b15e3ea

Reply:
Ah the updates LGTM! Let's merge with that change.